readme/terminal.md

```diff
@@ -269,193 +290,195 @@ The following commands are available in [command-line mode](#command-line-mode):
-	                                   (Basque (39%)), bs_BA (Bosnian (85%)), 
-	                                   bg_BG (Bulgarian (77%)), ca (Catalan 
-	                                   (61%)), hr_HR (Croatian (32%)), cs_CZ 
-	                                   (Czech (94%)), da_DK (Dansk (85%)), de_DE 
-	                                   (Deutsch (100%)), et_EE (Eesti Keel (76%)), 
-	                                   en_GB (English (UK) (100%)), en_US (English 
-	                                   (US) (100%)), es_ES (Español (95%)), eo 
-	                                   (Esperanto (44%)), fr_FR (Français (95%)), 
-	                                   gl_ES (Galician (50%)), it_IT (Italiano 
-	                                   (97%)), nl_BE (Nederlands (39%)), nl_NL 
-	                                   (Nederlands (97%)), nb_NO (Norwegian 
-	                                   (89%)), fa (Persian (38%)), pl_PL (Polski 
-	                                   (75%)), pt_PT (Português (91%)), pt_BR 
-	                                   (Português (Brasil) (88%)), ro (Română 
-	                                   (39%)), sl_SI (Slovenian (49%)), sv 
-	                                   (Svenska (68%)), tr_TR (Türkçe (92%)), 
-	                                   el_GR (Ελληνικά (93%)), ru_RU (Русский 
-	                                   (95%)), sr_RS (српски језик (75%)), zh_CN 
-	                                   (中文 (简体) (97%)), zh_TW (中文 (繁體) (91%)), 
-	                                   ja_JP (日本語 (97%)), ko (한국말 (97%)).
+	                                   Possible values: ar (Arabic), eu (Basque),
+	                                   bs_BA (Bosnian), bg_BG (Bulgarian),
+	                                   ca (Catalan), hr_HR (Croatian),
+	                                   cs_CZ (Czech), da_DK (Dansk),
+	                                   de_DE (Deutsch), et_EE (Eesti Keel),
+	                                   en_GB (English (UK)), en_US (English (US)),
+	                                   es_ES (Español), eo (Esperanto),
+	                                   fi_FI (Finnish), fr_FR (Français),
+	                                   gl_ES (Galician), id_ID (Indonesian),
+	                                   it_IT (Italiano), nl_BE (Nederlands),
+	                                   nl_NL (Nederlands), nb_NO (Norwegian),
+	                                   fa (Persian), pl_PL (Polski),
+	                                   pt_PT (Português),
+	                                   pt_BR (Português (Brasil)), ro (Română),
+	                                   sl_SI (Slovenian), sv (Svenska),  
+	                                   th_TH (Thai), vi (Tiếng Việt),
+	                                   tr_TR (Türkçe), el_GR (Ελληνικά),
+	                                   ru_RU (Русский), sr_RS (српски језик),
+	                                   zh_CN (中文 (简体)), zh_TW (中文 (繁體)),
+	                                   ja_JP (日本語), ko (한국말).
```

Comment:
It could very well be that these are generated.

Not sure. If not, I'm ok with removing the percentages here, since we already have them in the README.

@laurent22 what do you think?

Reply:
I added a bunch into here that were present in the README but not the old file (e.g. vi and id_ID) so if it is auto generated then there is something not syncing up unless they only show above a certain % completion?

Reply:
I don't think it is auto-synced, because every time I generate the translations, only the README is updated. So I think this is a good change. There's no reason to have the percentages in the terminal.md file as well.

I just wanted to confirm with Laurent.

Reply:
Yes, we shouldn't update this part manually as this whole section is updated with the content of `joplin help all`. So most likely the languages are out-dated but also other commands. So everything should be updated in one go (either in this pull request or later on).